app/src/main/java/com/google/droidjump/StartFragment.java

```diff
@@ -16,28 +16,28 @@
 
 package com.google.droidjump;
 
-import static androidx.navigation.Navigation.findNavController;
-
 import android.os.Bundle;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.Button;
 import android.widget.LinearLayout;
 import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentActivity;
 import com.google.android.material.floatingactionbutton.FloatingActionButton;
 import com.google.droidjump.models.LevelManager;
+import com.google.droidjump.models.NavigationHelper;
 
 /**
  * Displays Start Screen.
  */
 public class StartFragment extends Fragment {
-    private MainActivity activity;
+    private FragmentActivity activity;
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        activity = (MainActivity) getActivity();
+        activity = getActivity();
         View rootView = inflater.inflate(R.layout.start_screen, container, /* attachToRoot= */ false);
         Button playButton = rootView.findViewById(R.id.play_button);
         Button levelButton = rootView.findViewById(R.id.level_button);
```

Comment:
hmm, I just realized that these can be inlined. Anyway not for this PR, for future ones.

Reply:
Will create PR for this one.